docs/settings.md

```diff
@@ -28,12 +28,21 @@ For example, in case you want to run the app on port `5000`, just set the enviro
 
 ## Development
 
-* `--reload` - Enable auto-reload.
+* `--reload` - Enable auto-reload. Uvicorn supports two versions of auto-reloading behavior enabled by this option. There are important differences between them.
 * `--reload-dir <path>` - Specify which directories to watch for python file changes. May be used multiple times. If unused, then by default the whole current directory will be watched. If you are running programmatically use `reload_dirs=[]` and pass a list of strings.
+
+### Naive Reloading
+
+If Uvicorn _cannot_ load [watchgod](https://pypi.org/project/watchgod/) at runtime, it will periodically look for changes in modification times to any `*.py` file inside of its monitored directories. See the `--reload-dir` option. Note: _all_ `*.py` files, and _only_ `*.py` files will be monitored for changes. This is not configurable unless watchgod in installed.
```

Comment:
@Andrew-Chen-Wang, this adopts your suggested edit and removes what I perceive as extraneous words.

Reply:
```suggestion
If Uvicorn _cannot_ load [watchgod](https://pypi.org/project/watchgod/) at runtime, it will periodically look for changes in modification times to any `*.py` file inside of its monitored directories (see `--reload-dir`). Note: _all_ `*.py` files, and _only_ `*.py` files will be monitored for changes. This is not configurable unless watchgod in installed.
```

LGTM.

One concern is this quote: "Note: _all_ `*.py` files, and _only_ `*.py`". I thought only `.py` files which were imported would be subject to a reload. This wording implies any `.py` file will be reloaded, regardless of its location in the entire file system.

Reply:
I might be confused, but unless I'm misreading [the code](https://github.com/encode/uvicorn/blob/6885e9c6ec4d881ec5cf1464a07c619706ad8240/uvicorn/supervisors/statreload.py#L49-L52), I'm pretty sure it's _any_ `*.py` file inside the monitored directories that has a change to its `mtime`. In any event, I word-smithed this a little bit to make clearer that it was bound by the monitored directories.

Either way, we should definitely surface a clearer understanding of what actually happens, so I'm very grateful for the (continued) attention.